openmetadata-spec/src/main/resources/json/schema/api/data/createQuery.json

```diff
@@ -0,0 +1,69 @@
+{
+  "$id": "https://open-metadata.org/schema/api/data/createQuery.json",
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "title": "CreateQueryRequest",
+  "description": "Create Query Request",
+  "type": "object",
+  "javaType": "org.openmetadata.schema.api.data.CreateQuery",
+  "javaInterfaces": [
+    "org.openmetadata.schema.CreateEntity"
+  ],
+  "properties": {
+    "name": {
+      "description": "Name of a entity to which the query belong",
+      "$ref": "../../type/basic.json#/definitions/entityName"
+    },
+    "displayName": {
+      "description": "Display Name that identifies this query.",
+      "type": "string"
+    },
+    "description": {
+      "description": "Description of the query instance.",
+      "$ref": "../../type/basic.json#/definitions/markdown"
+    },
+    "owner": {
+      "description": "Owner of this entity",
+      "$ref": "../../type/entityReference.json",
+      "default": null
```

Comment:
Can this be both a team or a user? If so entityReference is okay.

Reply:
yes  we can have the owners for the SQL queries created through UI as users or teams